sqlite-android/src/main/jni/sqlite/Android.mk

```diff
@@ -26,7 +26,13 @@ sqlite_flags := \
 	-DSQLITE_ENABLE_RTREE=1 \
 	-DSQLITE_OMIT_BUILTIN_TEST \
 	-DSQLITE_OMIT_COMPILEOPTION_DIAGS \
-	-DSQLITE_DEFAULT_FILE_PERMISSIONS=0600
+	-DSQLITE_DEFAULT_FILE_PERMISSIONS=0600 \
+	-DSQLITE_DQS=0 \
+    -DSQLITE_DEFAULT_MEMSTATUS=0 \
```

Comment:
I checked, and I don't think the results of the sqlite function sqlite3_status() function are available by pragma or anything, so I don't think this is a breaking change. If I'm right, nice optimization